lib/plugins/sfDoctrinePlugin/lib/task/sfDoctrineBuildTask.class.php

```diff
@@ -130,7 +130,9 @@ protected function execute($arguments = [], $options = [])
             throw new InvalidArgumentException(sprintf("You must include one or more of the following build options:\n--%s\n\nSee this task's help page for more information:\n\n  php symfony help doctrine:build", join(', --', array_keys($this->getBuildOptions()))));
         }
 
-        if (self::BUILD_DB == (self::BUILD_DB & $mode)) {
+        $is_production = sfProjectConfiguration::getActive()->isProduction();
+
+        if (!$is_production && (self::BUILD_DB == (self::BUILD_DB & $mode))) {
```

Comment:
`BUILD_DB` is a shorthand for:
- drop the database wholesale
- rebuild a blank schema from the `config/schema.yml` file.

Absolutely unforgivable in production.